tfx/orchestration/pipeline.py

```diff
@@ -189,3 +190,76 @@ def components(self, components: List[base_node.BaseNode]):
     # has all its dependencies visited.
     if len(self._components) < len(deduped_components):
       raise RuntimeError('There is a cycle in the pipeline')
+
+  def get_fuseable_components(self):
+    """
+       Returns a list of fuseable Beam component subraphs in sorted topological
+       order. For example:
+
+       [[component_a, component_b],[component_c, component_d, component_e]]
+
+       Experimental, backward compatability not guaranteed.
```

Comment:
Can we move this to its own file?  Maybe tfx/orchestration/experimental/optimizations/beam_fusion.py?

CC: @ruoyu90 @zhitaoli

Reply:
Done.